src/components/header/PatientHeader.jsx

```diff
@@ -0,0 +1,215 @@
+              <em>Middle</em>
+            </span>
+          }
+  
+          <span>
+            <span className="PersonName-familyName">{person.personName.familyName}</span>
+            <em>Family Name</em>
+          </span>
+  
+          &nbsp;
+          <span className="gender-age">
+            <span>{person.gender === 'M' ? "Male" : "Female"}&nbsp;</span>
+            <span>
+              {person.age} year(s) ({dateFns.format(new Date(person.birthdate), 'DD[.]MMM[.]YYYY')})
+            </span>
+            <span
+              className="edit-info"
+              id="edit-patient-demographics"
+            >
+              <small>
+                <a href={`../../registrationapp/editSection.page?patientId=${patientId}&sectionId=demographics&appId=referenceapplication.registrationapp.registerPatient`}>Edit</a>
+              </small>
+            </span>
+            <a
+              className="contact-info-label expanded"
+              id="patient-header-contactInfo"
+              onClick={() => { this.toggleDetailsView(); }}
+              role="button"
+              tabIndex="0"
+            >
+              {showContactInfo ?
+                <span>Hide Contact Info <i className="toggle-icon icon-caret-up small" /></span> :
+                <span>Show contact info <i className="toggle-icon icon-caret-up small rotate180" /></span>
+              }
+            </a>
+          </span>
+  
+          <div className="firstLineFragments" />
+  
+          {showContactInfo &&
+          <div
+            className=""
+            id="contactInfoContent"
+          >
+            <div className="contact-info-inline">
+              <span>
+                {person.preferredAddress.display}
+                {person.preferredAddress.cityVillage && ` , ${person.preferredAddress.cityVillage}`}
+                {person.preferredAddress.stateProvince && `,${person.preferredAddress.stateProvince}`}
+                {person.preferredAddress.country && `,${person.preferredAddress.country}`}
+                {person.preferredAddress.postalCode && `,${person.preferredAddress.postalCode}`}
+                <em>Address</em>
+              </span>
+              <span className="left-margin">
+                <span id="coreapps-telephoneNumber">
+                  {attributes[0] ? attributes[0].value : ''}
+                </span>
+                <em>Telephone Number</em>
+              </span>
+                &nbsp;&nbsp;
+              <small
+                className="edit-info"
+                id="contact-info-inline-edit"
+              >
+                <a
+                  href={`../../registrationapp/editSection.page?patientId=${patientId}&sectionId=contactInfo&appId=referenceapplication.registrationapp.registerPatient`}
+                >
+                  Edit
+                </a>
+              </small>
+            </div>
+          </div>
+          }
+        </h1>
+      </div>
+    )
+  }
+
+  renderSecondLineFrangments() {
```

Comment:
I didn't notice this "note" functionality before, but I don't think we really need to include it at all in this basic header and can just pull this whole section out... does that make sense to you @dkayiwa ?